### PR TITLE
Fix for Expand/Colapse problem when GroupClick is bound.

### PR DESCRIPTION
**Pull request by AlexStefan**
### :sparkles: What kind of change does this PR introduce? (Bug fix, feature, docs update...)
Bug fix

### :arrow_heading_down: What is the current behavior?
When GroupClick is bound the group is not expanding/collapsing

### :new: What is the new behavior (if this is a feature change)?
When GroupClick is bound the group will still expand/collapse

### :boom: Does this PR introduce a breaking change?
No

### :memo: Links to relevant issues/docs
https://github.com/MvvmCross/MvvmCross/issues/2408

### :thinking: Checklist before submitting

- [x] All projects build
- [x] Follows style guide lines ([code style guide](https://github.com/MvvmCross/MvvmCross#code-style-guidelines))
- [ ] Relevant documentation was updated ([docs style guide](https://www.mvvmcross.com/documentation/contribute/mvvmcross-docs-style-guide))
- [ ] Nuspec files were updated (when applicable)
- [x] Rebased onto current develop

### CODE REVIEW

File: MvvmCross/Binding/Droid/Views/MvxExpandableListView.cs [hunk 1]
```diff
@@ -151,7 +151,11 @@ private void EnsureGroupClickOverloaded()
 
         private void GroupOnClick(object sender, GroupClickEventArgs e)
         {
-           ExecuteCommandOnGroup(GroupClick, e.GroupPosition);
+            if (IsGroupExpanded(e.GroupPosition))
```
Comment (Cheesebaron) [inline MvvmCross/Binding/Droid/Views/MvxExpandableListView.cs]: Is this change really needed, don't we just need to set `GroupClickEventArgs.Handled` property to true or false to signal whether we handled the event?

Comment (AlexStefan) [reply MvvmCross/Binding/Droid/Views/MvxExpandableListView.cs]: Tried to find a place to set that property but I could not find one as the args are already in there when the method is getting called. By looking around I saw that there is a difference over how we use ItemClick and GroupClick, but will look more into it tomorrow. (GroupClick exists in both base and Mvx when ItemClick exists only in Mvx, while we have ChildClick in base, and maybe something goes wrong around this?) Any other idea?

Comment (Cheesebaron) [reply MvvmCross/Binding/Droid/Views/MvxExpandableListView.cs]: Just as I suggested, it is enough to just add `e.Handled = false;` after `ExecuteCommandOnGroup(..)`.

**Comment by AlexStefan** [comment]
Not ok, working to remove the bad commits and apply your suggestion.
